public/geo/gb_counties_fips.json

```diff
@@ -0,0 +1,385 @@
+[
+    {"county_name":"Conwy Principal Area", "county_fips_code":"W06000003"}, 
```

Comment:
We're sticking with using US-ism like `fips` and `zctas` for I18N?  I know it's just in the code, but we should document what that means for other countries somewhere.

Reply:
SGTM.  I suppose you could add a comment to `zcta-county` to what exactly things map to.  For example, zctas = Local Authority Districts?

Reply:
Actually, I think we should have a separate branch just to remove all US-isms. And use something like "sub_region_2" instead of zcta and "sub_region_3_code" instead of fips. Otherwise each country will probably have their own names for the region levels, and it will quickly become super confusing. But in interest of making things work for now, I'd keep the US naming for consistency.